packages/app/src/app/pages/Sandbox/SignOutNoticeModal/index.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import { inject, observer } from 'app/componentConnectors';
+import { useOvermind } from 'app/overmind';
 
 import { Button } from '@codesandbox/common/lib/components/Button';
 import Row from '@codesandbox/common/lib/components/flex/Row';
 
 import { Container, Heading, Explanation } from './elements';
 
-function SignOutNotice({ signals }) {
+function SignOutNotice() {
```

Comment:
```suggestion
export const SignOutNotice = () => {
```

Let's change this to an arrow function and use a named export here. This will necessitate updating import references across the code base to replace the current default export, but this is our preferred way of handling imports/exports.

Reply:
changed to arrow function. Seems this component has not used yet, hence did not find any references across the code.